blocksuite/affine/shared/src/__tests__/helpers/affine-test-utils.ts

```diff
@@ -0,0 +1,115 @@
+import type { BlockModel, Store } from '@blocksuite/store';
+import { expect } from 'vitest';
+
+declare module 'vitest' {
```

Comment:
Maybe have a `@blocksuite/test-kit` package is more helpful than put it in `@blocksuite/affine-shared`?